src/Tribe/Views/V2/Hooks.php

```diff
@@ -60,6 +60,7 @@ protected function add_actions() {
 		add_action( 'tribe_events_pro_shortcode_tribe_events_after_assets', [ $this, 'action_disable_shortcode_assets_v1' ] );
 		add_action( 'updated_option', [ $this, 'action_save_wplang' ], 10, 3 );
 		add_action( 'the_post', [ $this, 'manage_sensitive_info' ] );
+		add_action( 'get_header', tribe_callback( Template\JSON_LD::class, 'print_single_json_ld' ) );
```

Comment:
We want to hook a method of the `Hooks` class, so that it can be easily removed by the `remove_(filter|action)` function.
This should change to:
```suggestion
		add_action( 'get_header', [ $this, 'print_single_json_ld' ] );
```

Reply:
In the `Hooks` class create a `print_single_json_ld` method that just relays the call to the `Template\JSON_LD::print_single_json_ld` class.